### DISCUSSION TITLE
Mini event : La grande loterie

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1642] [Nouvelles idées]
Ce serait une loterie organisée par Claire de Ville
Elle serait uniquement accessible sur la route des merveilles en partant en direction de Claire de Ville (donc HYPER rare)

A chaque fois qu'on tombe dessus, on peut jouer 100 gold afin de tenter de gagner le jackpot.

Au départ, le jackpot vaut 5 gemmes, et gagne 1 gemme à chaque fois qu'un joueur joue à la loterie.

La proba de gagner à la loterie est extrêmement faible, mais y'a un gros reward.

Si quelqu'un gagne la loterie, alors elle se reset à sa valeur de base.

Cela encouragerait les joueurs à venir se balader dans la zone afin de tenter de récupérer le gros lot

**Comment by BastLast** [comment]
Les gemmes doivent rester mission related imo

**Reply by romain22222** [reply]
je me disais que ca pouvait être sympa, mais du coup on peut remplacer par 5 gemmes -> 1000 gold, 1 gemme -> 100 gold

**Comment by romain22222** [comment]
je relance l'idée, j'aimerais voir vos avis

**Comment by KirIcare** [comment]
Perso je trouve ça intéressant, juste pour visualiser le jackpot on pourra faire une commande genre /jackpot ou /loterie ou le seul moyen de le connaître c'est d'aller à clair de ville via routes des merveilles, de tomber sur le mini event et seulement à ce moment là le montant est affiché ?

**Comment by QQtin3** [comment]
L'idée est super intéressante, je trouve cependant que la remarque de Draft juste, pourquoi ne pas se contenter de rajouter 100 pièces si le joueur ne gagne pas ?

**Reply by BastLast** [reply]
c'est plus drole si le gagnant rafle la thune de tout le monde ptdr

**Reply by KirIcare** [reply]
Je le pensais comme ça ouais
Comme lorsque tu fais 6 à la bonne paye
Le coup du trajet le moins fréquenté est top, c'était l'idée initiale de Romain je pense, donner envi de s'aventurer sur le côté droit de la map

**Reply by romain22222** [reply]
complètement exact, je vise un endroit jamais exploré et qui soit suffisament logique afin d'inviter les joueurs à tester d'autres endroits sur la map

**Comment by BastLast** [comment]
peut-on réfléchir à placer ce mini évent sur le trajet où il y a le moins de monde ? histoire que ça encourage les gens à se rendre dans cette zone ?

**Comment by BastLast** [comment]
Y a une problèmatique sur laquelle on doit se pencher, bcp de bots se sont fait ban du repo des bots parce qu'ils avaient un système de loterie, peut etre ya un autre concept positif pr les joueurs à faire des les zones à faible attrait qui ne soit pas clairement de la loterie en attendant que ça soit plus clair du coté des tos ?

**Comment by BastLast** [comment]
aussi je suis chaud pr que ce genre de tâche soit concu pr avoir un temps de dev minimal parce que ya beaucoup de features importantes qui arrivent et que si c trop long à faire ça sera placé loin dans le backlog

**Comment by romain22222** [comment]
quand on dit lotterie, c'est juste un truc du genre

"OUECH Y A UN GROS LOT A GAGNER DE XXX GOLDS, T AS UNE PTITE CHANCE DE GAGNER MAIS FAUT METTRE 100 GOLDS, ALORS TU VEUX JOUER ?

Oui
Non
"

Si c'est oui, alors tirage random (genre chance d'un mythique ou autre) et si ca gagne, ca récup la mise et ca la remet à 1000, sinon on paye la lotterie 100

Si non, alors "BYE BYE TROUILLARD LE MEC JOUE PAS CE CASOS"

**Comment by romain22222** [comment]
genre vraiment en terme de code, c'est que dalle

**Comment by romain22222** [comment]
même un nouveau contrib peut le faire c'est dire

**Reply by Easygamer01s** [reply]
Bonsoir,

Je vois l'espérance de gain de cet event et perso ça ne me ferait pas changer de zone :)  L'évent du dédale dispose déjà de meilleurs stat :) 3000 avec une proba qui doit être de l'ordre de 1/12 ou 1/16, si on prend en compte les events de la même localisation :)
La rotation à gauche de la zone resterait aussi plus intéressante que le chemin vers l'event :) 
Perso ce qui me ferait bouger c'est des drops ;)

**Reply by Easygamer01s** [reply]
> Bonsoir,
> 
> Je vois l'espérance de gain de cet event et perso ça ne me ferait pas changer de zone :)  L'évent du dédale dispose déjà de meilleurs stat :) 3000 avec une proba qui doit être de l'ordre de 1/12 ou 1/16, si on prend en compte les events de la même localisation :)
> La rotation à gauche de la zone resterait aussi plus intéressante que le chemin vers l'event :) 
> Perso ce qui me ferait bouger c'est des drops ;)

Ou des variations de prix dans le shop ;)

**Reply by romain22222** [reply]
je précise que c'est pas un event, mais un **mini-event**, ca veut donc dire qu'il tombera entre les events, donc sur le chemin entre les deux destinations, de plus le taux d'apparition du mini event sur ce dit chemin sera plus forte pour augmenter le nombre de personne venant y circuler et ainsi rendre le chemin attrayant

de plus, il se peut qu'il y ait une grosse somme de sauvegardée, ca veut donc dire que par moments ce sera insane

**Reply by Easygamer01s** [reply]
En mini event ça change la donne ;) tu parles de gros gain, on pourrait imposer une loi de probabilité comme une Pareto, pour avoir des évènements extrêmes :) la tangente hyperbolique dans les gains en trophée j'ai trouvé ça top :)

**Comment by Procion06** [comment]
je trouve l'idée très bonne mais si vous pensez qu'un mini event qu'on n'est pas sûr d'avoir et qui a une infime chance de drop va contrebalancer la mauvaise répartition des récompenses sur la map c'est sous-estimer le bon sens des joueurs :)
